### PR TITLE
KB Corrections

### PR DESCRIPTION
**Pull request by ignotus666**
# Does this need translation?

- [ ] Yes 
- [* ] No 

# Changes

A few corrections to documents in _posts.

### CODE REVIEW

File: _posts/2021-01-05-Jamulus-Sound-Devices.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Just [edit this page and wait for an approval](https://github.com/jamulussoftwar
 
 ## Audio devices known to work with Jamulus
 
-Your device must support a sample rate of *48kHz (DVD quality)* - which many devices do - and should allow you to listen to the computer’s audio signal. On Windows it’s recommended to choose a device with a *native ASIO driver* - not ASIO4ALL.
+Your device must support a sample rate of *48 kHz (DVD quality)* - which many devices do - and should allow you to listen to the computer’s audio signal. On Windows it’s recommended to choose a device with a *native ASIO driver* - not ASIO4ALL.
```
Comment (pljones) [inline _posts/2021-01-05-Jamulus-Sound-Devices.md:22]: Are we aiming to be consistent with "48000 Hz" vs "48 kHz"?  If both are to be used, is there guidance on when each is appropriate?

File: _posts/2021-01-05-Jamulus-Sound-Devices.md [hunk 2]
```diff
@@ -180,7 +180,7 @@ _More testing required._
 
 **Windows**: Works great. If you have issues with the buffer size going to 136, download the updated driver version `4.64.15.598` from [http://beta.focusrite.com/](http://beta.focusrite.com/)
 
-**macOS**: Works great. (measured interface latency with Oblique Audio RTL Utility, 48000hz, 64 samples and cable from output to input on Mac OS X: around `7.5ms`. Note this is **only the interface latency** not the latency in Jamulus)
+**macOS**: Works great. (measured interface latency with Oblique Audio RTL Utility, 48000 hz, 64 samples and cable from output to input on Mac OS X: around `7.5 ms`. Note this is **only the interface latency** not the latency in Jamulus)
```
Comment (pljones) [inline _posts/2021-01-05-Jamulus-Sound-Devices.md]: "Hz" not "hz"

File: _posts/2021-01-05-Jamulus-Sound-Devices.md [hunk 3]
```diff
@@ -238,7 +238,7 @@ _More testing required._
 
 **[Presonus Studio 24C](https://www.presonus.com/products/Studio-24c)** USB digital audio interface
 
-**Windows**: Works great with provided ASIO driver. (measured interface latency with Oblique Audio RTL Utility, 48000hz, 64 samples and cable from output to input on Mac OS X Big Sur: `4.1ms`. Note this is **only the interface latency** not the latency in Jamulus)
+**Windows**: Works great with provided ASIO driver. (measured interface latency with Oblique Audio RTL Utility, 48000 hz, 64 samples and cable from output to input on Mac OS X Big Sur: `4.1 ms`. Note this is **only the interface latency** not the latency in Jamulus)
```
Comment (pljones) [inline _posts/2021-01-05-Jamulus-Sound-Devices.md]: "Hz" not "hz"

File: _posts/2021-01-05-Jamulus-Sound-Devices.md (hunk 2 above)
Comment (ignotus666) [reply _posts/2021-01-05-Jamulus-Sound-Devices.md]: Corrected

File: _posts/2021-01-05-Jamulus-Sound-Devices.md (hunk 3 above)
Comment (ignotus666) [reply _posts/2021-01-05-Jamulus-Sound-Devices.md]: Corrected

File: _posts/2021-01-05-Jamulus-Sound-Devices.md (hunk 1 above)
Comment (ignotus666) [reply _posts/2021-01-05-Jamulus-Sound-Devices.md:22]: My preference would be to stick to 48 kHz but I agree it might be best to use one or the other.

**Review by ann0see — APPROVED** [review]
Thank you for the corrections. Since this file is part of the KB, it is and was rather messy and inconsistent (but we know that due to the fact that it's an open document which can be edited by "everyone").

**Pull request merged**